joomla/joomla-cms
# Catching a throw from JLanguage::parseXMLLanguageFile

If you have a missing language manifest file (eg you deleted it accidentally), you will not be able to login to backend anymore because it will show you a big nice error.

### Summary of Changes
Catching the Exception from `JLanguage::parseXMLLanguageFile` in `JLanguageHelper:.getInstalledLanguages` and show a useful warning instead.

### Testing Instructions
1. Install a second language
2. Delete the language folders. Don't uninstall the language using the Extension Manager.
3. Try editing your profile or log out from backend and try log in again.

With current staging, this will get you a big error and you can't do anything anymore.
With this patch you will get a warning "Could not parse language ArabicUnitag (ar-AA)" which should be more helpful and lets you proceed.

### Documentation Changes Required
None
